client/jetpack-cloud/sections/agency-dashboard/hooks.tsx

```diff
@@ -453,3 +461,40 @@ export const useShowVerifiedBadge = () => {
 
 	return { verifiedItem, handleSetVerifiedItem };
 };
+
+export const useLicenseLightbox = () => {
+	const dispatch = useDispatch();
+	const { data: products } = useProductsQuery();
+	const selectedLicense = useSelector( getSelectedLicenseInfo );
+
+	const productSlug = selectedLicense ? getProductSlugFromProductType( selectedLicense ) : null;
+
+	return useMemo( () => {
+		return {
+			license:
+				productSlug && products
+					? products.find( ( product ) => product.slug === productSlug )
+					: null,
+			onActivate: () => {
+				if ( selectedLicense ) {
+					dispatch(
+						recordTracksEvent( 'calypso_jetpack_agency_dashboard_licenses_select', {
```

Comment:
Could we please use the `useJetpackAgencyDashboardRecordTrackEvent` hook here, which is already there in this file?